magnet_derive/src/codegen_field.rs

```diff
@@ -51,10 +51,17 @@ fn impl_bson_schema_named_fields(
 ) -> Result<TokenStream> {
     let properties = &field_names(attrs, &fields)?;
     let defs: Vec<_> = fields.iter().map(field_def).collect::<Result<_>>()?;
+    let doc = doc_meta(&attrs).and_then(|doc| meta_value_as_str(&doc).ok());
```

Comment:
Please do not use `Result::ok()` here – attribute parsing errors are meant to be propagated upward.